### ISSUE TITLE
Please give us the proper italic cuts!

### ISSUE THREAD

**Issue by timwoolliscroft** [google/fonts#3104]
I understand that for web design, italics are not strictly needed due to the variable font. But for designers working in the entire adobe suite, rolling out a brand that needs to be used by MANY people, it's a complete nightmare in terms of consistency.

Please give us all the set italic cuts of inter too!

Cheers,

Tim

**Comment by davelab6** [comment]
IIRC These aren't included because upstream they are in a "slant" axis and that isn't supported in CSS and browsers correctly. We could onboard them as a separate italic font with the same range of other axes, and I believe @arrowtype did discuss this with @rsms, but I think the decision was not to do it. Maybe we should revisit that.

**Comment by timwoolliscroft** [comment]
Hi Dave,

I think this would be something well worth revisiting. We've actually uncovered more issues with the Variable face when used on a Windows machine. It simply doesn't work in either MS Office or Adobe CC, it just shows up as multiple versions of Regular. If we had set cuts in italic, we think it would all work. Our client is a pretty big organisation, and we're thinking we might have to swap to a different font at the moment as it's just not looking like a viable option.

(Side note, we also discovered that the programming of the font in MS word is a little off – meaning the angle of the italic is way too steep. We think you currently have it set to 15°, where actually it needs to be 9° to closer match the actual Italics seen in the Variable face).

I really hope this is something we'll be able to figure out!

Cheers

Tim

**Comment by arrowtype** [comment]
1. @timwoolliscroft all of us agree that Italics are useful in typography! For now, your best bet is to download the full release from the official Inter website: https://rsms.me/inter/.

2. Yes, I think this is probably worth revisiting. As @davelab6  says, there are two ways Italics can be made to work in a variable font: the `Slant` axis (which gives smooth control over degrees of angle) and the `Italic` axis (which is a simpler `0-1`, binary axis). The Slant axis isn’t yet supported in the Safari browser with the CSS `font-style: italic`. And then, most browsers don’t yet fully support other parts of the CSS & OpenType spec, such as `font-style: oblique Xdeg;`. But yes, I think that serving Italic fonts which at least most users will see (in Chrome & Firefox) is better than serving fonts without italics, in which case users in all browsers will end up with faux-italics.

I have a [test](https://arrowtype.github.io/vf-slnt-test/faux-slnt.html) of this, which fails in Safari, but works in Chrome (and Firefox). When it fails, you can see the words not overlapping, whereas when it succeeds, the overlap is perfect:

![image](https://user-images.githubusercontent.com/45946693/108562538-06eda880-72ce-11eb-8fe0-ea7231698f09.png)

![image](https://user-images.githubusercontent.com/45946693/108562524-fdfcd700-72cd-11eb-8a76-f0401ca27a86.png)

The Italic instances are not included in the Google Fonts download, but I think they really should be:
![image](https://user-images.githubusercontent.com/45946693/108561006-88900700-72cb-11eb-8f5c-33b6da153cfe.png)

> (Side note, we also discovered that the programming of the font in MS word is a little off – meaning the angle of the italic is way too steep. We think you currently have it set to 15°, where actually it needs to be 9° to closer match the actual Italics seen in the Variable face).

Currently, there are still lots of issues with using variable fonts in desktop apps. I recommend using the Static fonts for 99% of users in 99% of Desktop use cases, and keeping the variable font for the website and for developers/designers who want to do some more advanced work with it.

**Comment by timwoolliscroft** [comment]
Hi, thanks for getting back to me.

We've tried using the Foundry download as you've suggested, but their static fonts also have issues in MS Office on PC. Their bold face doesn't actually display as bold within Word, but when exported as a PDF the bold shows up. Not ideal as it could mean random bits of bold creeping in without the user being aware.

Plus the fact that anyone wanting to grab Inter quickly is probably going to search for it and see the Google link first. We'd ideally go with your version if it had the italics.

Do you think there is any likelihood of the italic static version of Inter being added any time soon? We're currently caught between a rock and a hard place of not wanting to replace the font all together, as it's going to reflect really poorly on us as a design agency, but we feel we can't recommend either the Foundry or Google versions as they currently stand.

Cheers,

Tim

**Comment by tiroj** [comment]
> there are two ways Italics can be made to work in a variable font

Three ways: make the roman and italic as separate variable fonts.

**Comment by timwoolliscroft** [comment]
It seems a most variable fonts in Google's library have them all included in the one typeface, few examples below.

https://fonts.google.com/specimen/Newsreader?preview.text_type=custom&vfonly=true
https://fonts.google.com/specimen/Roboto+Mono?preview.text_type=custom&vfonly=true
https://fonts.google.com/specimen/Raleway?preview.text_type=custom&vfonly=true
https://fonts.google.com/specimen/Playfair+Display?preview.text_type=custom&vfonly=true

**Comment by arrowtype** [comment]
Good point, @tiroj! There could indeed be a separate variable font for Italic styles (`slnt -10`), until browsers properly support this.

**Comment by arrowtype** [comment]
@timwoolliscroft Those include Roman & Italic in the same typeface (as in type family), but no in the same _font_ (as in font file).

![image](https://user-images.githubusercontent.com/45946693/108566752-eb39d080-72d4-11eb-9584-047bb2d6d6d7.png)

John is suggesting that Google Fonts could take `slnt`-axis fonts like Inter (and Recursive) and split them into two font files, in a similar way, until browsers fully support the `slnt` axis with the `font-style` CSS property. It would take a bit of work, but seems like a good idea to me.

**Comment by timwoolliscroft** [comment]
Gotcha @arrowtype! 

Yes, as long as this would create static italic styles to use on desktop, then it would solve all of our problems.

Being honest, do you think this is something likely to happen any time soon? Already appreciate you all taking the time to respond on this.

Cheers!

**Comment by arrowtype** [comment]
Just guessing here, but...

I believe that the MS Word bug from Inter could be fixed relatively quickly – I think I have a good guess at why it’s broken in MS Word, as I have previously had & fixed a similar issue in my project, Recursive. Also, the Inter project itself is controlled by a single individual, Rasmus, who tends to be relatively responsive.

However, adding Italic versions to Google Fonts would likely take some more time. More people are involved, and they have more considerations to make before changing anything on the site (e.g. they wouldn’t want to unintentionally introduce a new problem while fixing this one). But, this is all just speculation. @davelab6 would be able to say more clearly whether this is something that could be changed quickly or not.

**Comment by timwoolliscroft** [comment]
@arrowtype 

Is this the bug fix you're talking about: https://github.com/arrowtype/recursive/issues/373#issuecomment-670071028 

I'm afraid at this point the fix has gone a little over my head, as I'm completely lost in MS Office. I've shown the comment to our specialist to see if she can work it out. 

Also, is this fix something that every new user will have to do, or is it just a one off when setting up the templates?

**Comment by arrowtype** [comment]
> Is this the bug fix you're talking about: arrowtype/recursive#373 (comment)

Yes, I think they may be related, though I’m not positive.

This definitely isn’t something that every user would have to do. The fix would be to the internal metadata of the fonts, to make sure that the correct things are set to tell windows "This font is Regular, this other font is Bold" etc.

I think that probably, the `fix-fsselection` function of https://github.com/googlefonts/gftools/ would probably work, though it would have to be tested to know for sure.

**Comment by timwoolliscroft** [comment]
I think we've found the back end issue with the Foundry version. https://github.com/rsms/inter/issues/352#issuecomment-782474827 
Just need to get them to change the metadata and get a new version up.

Also if there is anything else I can do to get the wheels in motion to get static italic cuts approved by Google, please let me know. It's still something I'm keen on getting my hands on!

Cheers,

Tim

**Comment by tiroj** [comment]
Some background: when the OTvar working group was working on the definitions of the registered variation axes, one of the technical requirements that was identified was that because of the way the new STAT table was structured, there needed to be a corresponding fvar axis record for each STAT record. This is the primary reason for the existence of the ital axis registration. Of course, having decided to register this axis, the working group decided to define it in such a way that it _could_ be usable, i.e. not simply as a stub axis record, but as something that could be implemented in a variable font. In talking with other members of the working group, I never had the impression that this implementation was recommended or even that it was expected to be common. Most of the hypothetical use cases for the ital axis involved using it as something like a stylistic set feature, i.e. to change the appearance of _some_ glyphs in a font to a more cursive shape, probably in combination with the slnt axis.

I think the number of font families in which it makes sense to bundle the italics into a single variable font with the romans is probably pretty small. I really don’t see much benefit to doing so.
